foundation/foundation-lineage/foundation-data-lineage/foundation-data-lineage-java/src/main/java/com/boozallen/aissemble/data/lineage/DatasetFacet.java

```diff
@@ -57,7 +57,7 @@ public class DefaultDatasetFacet extends OpenLineage.DefaultDatasetFacet {
         private final URI schemaUrl;
 
         public DefaultDatasetFacet(String schemaUrl, URI producer) {
-            super(producer);
+            super(producer, null);
```

Comment:
Q: Is this due to a change in the parent class? What parameter as we passing as null?

Reply:
it's a new param added to the constructor, which is used to mark for delete facet